pkg/policy/validate.go

```diff
@@ -1483,14 +1483,23 @@ func jsonPatchOnPod(rule kyverno.Rule) bool {
 	return false
 }
 
-func validateKinds(kinds []string, mock bool, client *dclient.Client, p kyverno.ClusterPolicy) error {
+// validateKinds verifies if an API resource that matches 'kind' is valid kind
+// and found in the cache, returns error if not found
+func validateKinds(kinds []string, mock bool, client *dclient.Client, p kyverno.ClusterPolicy) (bool, error) {
```

Comment:
How's the first return boolean being used?

Reply:
Not anymore, forget to cleanup